### PR TITLE
added support for python3 __pycache__ folders

### PR DESCRIPTION
**Pull request by lacion**

### CODE REVIEW

**Comment by lacion** [comment]
excuse the second pull request for the review, had a mischief with my git config.

File: pyclean.fish [hunk 1]
```diff
@@ -7,5 +7,5 @@ function pyclean -d "Recursively clean directory from .pyc and .pyo files"
     set path2CLEAN $argv
   end
 
-  find $path2CLEAN -name "*.py[co]" -type f -delete
+  find $path2CLEAN -type f -name "*.py[co]" -delete -or -type d -name "__pycache__" -delete
```
Comment (bpinto) [inline pyclean.fish:10]: Will the `-or` condition check on both folders or only the first?

**Comment by lacion** [comment]
after running it it got rid of both pyc / pyo files and **pycache** folder

**Comment by bpinto** [comment]
:+1:

**Comment by scorphus** [comment]
Good job, @lacion! Thanks!

@bpinto: take my non-waited :+1:

**Comment by bpinto** [comment]
:P

**Pull request merged**